common/utils.py

```diff
@@ -103,6 +103,8 @@
         "what_is_your_name",
         "where_are_you_from",
         "who_made_you",
+        "move_forward",
+        "move_backward",
```

Comment:
they are not necessy here because you do not use them for dff_intent_responder_skill

Reply:
forgot to remove earlier, removed